composer.json

```diff
@@ -18,8 +18,10 @@
             "helpers/json.php"
         ]
     },
-    "require": {},
+    "require": {
+        "php": ">=7.3"
+    },
     "require-dev": {
-        "phpunit/phpunit": "^7"
+        "phpunit/phpunit": "^9.5"
```

Comment:
The PHPUnit 9.5 runs on PHP 7.3+, so we can use it right now

https://phpunit.de/supported-versions.html